src/lib/accessibility/Behaviors/Button/ButtonBehavior.ts

```diff
@@ -1,5 +1,11 @@
 import { Accessibility } from '../../interfaces'
 
+/**
+ * @description
+ *  Adds role='button' if element type is other than 'button'. This allows the screen readers handle component as button.
+ *  Adds attribute 'aria-disabled=true' based on the property 'disabled'.
```

Comment:
I would change ' This allows the screen readers handle component as button.' to 'This allows the screen readers to handle the component as button.'